test/simulator/framework/includes/mock_qtouch.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 Shift Cryptosecurity AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef _QTOUCH_H_
+#define _QTOUCH_H_
+
+#include <stdint.h>
+
+void __wrap_qtouch_process(void);
+
+uint8_t __wrap_qtouch_get_scroller_is_active(uint16_t sensor_node);
+
+uint16_t __wrap_qtouch_get_scroller_position(uint16_t sensor_node);
+
```

Comment:
wrap stuff not needed, it's only used in unit tests to replace functions with the `-Wl,wrap=...` linker args